pretext/project/__init__.py

```diff
@@ -342,24 +339,60 @@ def build(
 
             log.info(f"Preparing to build into {self.output_dir_abspath()}.")
 
-            build.build(
-                self.format,
-                self.source_abspath(),
-                self.publication_abspath(),
-                self.output_dir_abspath(),
-                self.output_filename,
-                self.stringparams,
-                custom_xsl=custom_xsl,
-                xmlid=xmlid,
-                zipped=self.compression is not None,
-                project_path=self._project.abspath(),
-                latex_engine=self.latex_engine,
-                executables=self._project._executables.dict(),
-                # TODO: what if this isn't defined? Should we have a default in the project file instead?
-                braille_mode=self.braille_mode,
-            )
+            if self.format == "html":
+                core.html(
+                    xml=self.source_abspath(),
+                    pub_file=self.publication_abspath().as_posix(),
+                    stringparams=self.stringparams,
+                    xmlid_root=xmlid,
+                    file_format="html",
+                    extra_xsl=custom_xsl,
+                    out_file=None,
+                    dest_dir=self.output_dir_abspath().as_posix(),
+                )
+                # For codechat:
+                project_path = utils.project_path(self.source_abspath())
```

Comment:
@bjones1, could you look over the next few lines to make sure this is doing what it needs to?  I wasn't sure what the best way to do the project_path in the refactor (might it just be `Path().resolve()`?)